src/MyMessage/index.tsx

```diff
@@ -0,0 +1,176 @@
+import React, { useState } from 'react';
+
+import { Props } from './props';
+import { styles } from './styles';
+
+import { Dot } from '../Dot';
+
+import { isImage, getFileName } from '../util/file';
+import { formatTime, getDateTime } from '../util/dateTime';
+
+import { Row, Col, setConfiguration } from 'react-grid-system';
+
+setConfiguration({ maxScreenClass: 'xl' });
+
+export const MyMessage: React.FC<Props> = ({
+  lastMessage = null,
+  message,
+  nextMessage = null,
+  chat = null,
+  isSending = false,
+}) => {
+  const [hovered, setHovered] = useState<boolean>(false);
+
+  const topRightRadius =
+    !lastMessage || lastMessage.sender_username !== message.sender_username
+      ? '1.3em'
+      : '0.3em';
+  const bottomRightRadius =
+    !nextMessage || nextMessage.sender_username !== message.sender_username
+      ? '1.3em'
+      : '0.3em';
```

Comment:
some repeated logic- maybe create a helper function

Reply:
The logic is a bit different, I typically enforce a refactor 3 or more times. 

~1.5 times is typically fine with me

Reply:
ya I also find giving something a name makes the code easier to read but will leave up to you